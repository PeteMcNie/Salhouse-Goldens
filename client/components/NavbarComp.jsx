import React from 'react'

import { Navbar, Nav, NavDropdown } from 'react-bootstrap'

const NavbarComp = () => {
  return (
    <>
      <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark" fixed='top'>
        <Navbar.Brand href="/">
          <img
            alt="Salhouse Goldens logo, phoenix and kula laying down in the bushes looking at camera"
            src="images/logo.jpeg"
            className="shlogo d-inline-block align-top"
          />
        </Navbar.Brand>

        <Navbar.Toggle aria-controls="responsive-navbar-nav" />

        <Navbar.Collapse id="responsive-navbar-nav">
          <Nav className="ml-auto">
            <Nav.Link href="/">Home</Nav.Link>

            <NavDropdown title="Our Goldens" id="collasible-nav-dropdown">
              <NavDropdown.Item href="#action/3.1">Phoenix</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.2">Kula</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.3">Shadow</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.4">Sherry</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.5">Autumn</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.6">Tuppence</NavDropdown.Item>
              <NavDropdown.Item href="#action/3.7">Siren</NavDropdown.Item>
            </NavDropdown>

            <NavDropdown title="Grooming Your Golden" id="collasible-nav-dropdown">
              <NavDropdown.Item href="#action/4.1">Basic Gromming Tips</NavDropdown.Item>
              <NavDropdown.Divider />
              <NavDropdown.Item href="#action/4.2">Grooming Service</NavDropdown.Item>
              <NavDropdown.Item href="#action/4.3">Testimonials and photos</NavDropdown.Item>
            </NavDropdown>

            <Nav.Link href="mylifewithanimals">My Life With Animals</Nav.Link>

            <NavDropdown title="About Us" id="collasible-nav-dropdown">
              <NavDropdown.Item href="/aboutus">What we are about</NavDropdown.Item>
              <NavDropdown.Item href="/aboutus">Where &apos;Salhouse&apos; comes from</NavDropdown.Item>
            </NavDropdown>

            <Nav.Link href="#links">Links</Nav.Link>
            <Nav.Link href="#contact">Contact</Nav.Link>
          </Nav>
        </Navbar.Collapse>
      </Navbar>
    </>
  )
}

export default NavbarComp
