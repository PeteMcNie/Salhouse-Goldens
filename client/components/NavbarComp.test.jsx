import React from 'react'
import { render } from '@testing-library/react'

import NavbarComp from './NavbarComp'

test('Navbar component to match snapshot', () => {
  const { container } = render(<NavbarComp />)
  expect(container).toMatchSnapshot()
})
