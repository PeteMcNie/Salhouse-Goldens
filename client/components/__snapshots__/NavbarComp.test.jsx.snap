// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component to match snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo1.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
</div>
`;
